Driver_Timer #2

```diff
@@ -0,0 +1,285 @@
+#include <FastLED.h>
+
+#define NUM_LEDS 60
+#define LED_PIN 2
+#define DATA_PIN 7
+
+void showStrip() {
+FastLED.show();
+}
+
+void setAll(byte red, byte green, byte blue) {
+  for(int i = 0; i < NUM_LEDS; i++ ){
+  setPixel(i, red, green, blue);
+    
+  }
+  showStrip();
+}
+
+// The command above says to set all of the pixels to a 
+//combination of a red byte, a green byte, and a blue byte
+
+CRGB leds[NUM_LEDS];
+void setup() {
+FastLED.addLeds<NEOPIXEL, LED_PIN>(leds, NUM_LEDS);
+
+Serial.begin(115200);
+}
+
+
+void setPixel(int Pixel, byte red, byte green, byte blue){
+leds[Pixel].r = red;
+leds[Pixel].g = green;
+leds[Pixel].b = blue;
+
+//a byte has two place values, each with 15 different numbers:
+//1 = 1
+//2 = 2
+// ... 
+//9 = 9
+//a = 10
+//b = 11
+//c = 12
+// ... 
+//f = 15
+//so 0xff means 15, 15
+//the byte system is a base 15 system
+
+}
+
+//This is the same command as setAll, but with specific pixels.
+//It takes a red, green, and blue byte.
+
+void loop() {
+  
+  
+  //set the color of all of the LEDS to GREEN
+  //time in auto
+  
+  setAll(0x00, 0xff, 0x00);
+  delay(1000);
+
+  //15
+  
+  leds[60] = CRGB::Black;
+  FastLED.show();
+  leds[59] = CRGB::Black;
+  FastLED.show();
+
+  leds[0] = CRGB::Black;
```

Comment:
I am not sure this would occur at the same second... Wouldn't this be a delay? I may think we may want to do it on a different thread?